epochStart/errors.go

```diff
@@ -211,3 +211,9 @@ var ErrNilLatestStorageDataProvider = errors.New("nil latest storage data provid
 
 // ErrNilShuffler signals that a nil shuffler was provided
 var ErrNilShuffler = errors.New("nil nodes shuffler provided")
+
+// ErrNotEnoughMinNumConnectedPeers signals that config is invalid for num of connected peers
+var ErrNotEnoughMinNumConnectedPeers = errors.New("not enough min num of connected peers from config")
```

Comment:
ErrNotEnoughNumConnectedPeers and  "not enough num connected peers from config"

Reply:
done